src/Make.inc

```diff
@@ -40,17 +40,17 @@ else ifeq ($(OS), Darwin)
 
   LIB_FULL_VERSION := libblastrampoline.$(LBT_SOVERSION_MAJOR).$(LBT_SOVERSION_MINOR).$(LBT_SOVERSION_PATCH).dylib
   LIB_MAJOR_VERSION := libblastrampoline.$(LBT_SOVERSION_MAJOR).dylib
-  SONAME_FLAG := -Wl,-install_name=@rpath/$(LIB_FULL_VERSION)
+  SONAME_FLAG := -install_name @rpath/$(LIB_MAJOR_VERSION)
```

Comment:
You can tell this wasn't set at all since the syntax was wrong and would have caused a linking error :stuck_out_tongue: